Tests/Serializer/BaseSerializerHandlerTest.php

```diff
@@ -69,6 +69,34 @@ public function testGetSubscribingMethods()
         $this->assertSame($methods, $expectedMethods);
     }
 
+    public function testSetFormats()
```

Comment:
It's good to check you did not break anything, so maybe let's indeed keep it, with a different name. Since the values are still hardcoded, it makes sense.